pytorch_to_returnn/torch/nn/modules/module.py

```diff
@@ -636,6 +636,9 @@ def check_call_returnn_outputs_to_prev_torch(call: CallEntry, *, update_ops: Lis
       returnn_output_np = returnn_output_np_.transpose(*[
         returnn_output_tensor_entry.returnn_axis_from_torch_axis[i] for i in range(returnn_output_np_.ndim)])
       torch_out_np = torch_mod_call.orig_outputs_flat[out_idx].detach().cpu().numpy()
+      assert torch_out_np.shape == tuple(int(i) for i in returnn_output_tensor_entry.tensor().shape), (
+        f"torch shape of TensorEntry does not match real torch shape: "
+        f"{torch_out_np.shape} vs. {returnn_output_tensor_entry}")
```

Comment:
The message should also include `returnn_output_tensor_entry.tensor().shape`.

Reply:
It's part of what is printed now, but we can also reduce it to only `returnn_output_tensor_entry.tensor().shape` if you prefer.

Reply:
No, both together are good.

Reply:
Where is `returnn_output_tensor_entry.tensor().shape`?

Reply:
In `TensorEntry.__repr__`